content/blog/announcing_kubeflow_0.3.md

```diff
@@ -0,0 +1,156 @@
+* The object detection [example-](https://github.com/kubeflow/examples/commits/master/object_detection) illustrates how to leverage GPUs for online and batch inference.
+*   The financial time series prediction example illustrates how to leverage Kubeflow for time series analysis ([blog](https://blog.ml6.eu/using-kubeflow-for-analyzing-financial-time-series-part-i-18580ef5df0b), [code](https://github.com/kubeflow/examples/tree/master/financial_time_series)).
+
+
+**Kubebench**
+
+Benchmarking is critical to identifying performance bottlenecks that slow development or waste expensive resources. To address these needs, Cisco has provided an initial release of [Kubebench](https://github.com/kubeflow/kubebench), a framework for benchmarking ML workloads on Kubeflow.
+
+**TFX**
+
+The 0.3 Kubeflow Jupyter images ship with [TF Data-Validation](https://github.com/tensorflow/data-validation). TF Data-Validation is a library for exploring and validating machine learning data.
+
+And many more. For a complete list of updates, see the 0.3 [Change Log](https://github.com/kubeflow/kubeflow/blob/master/CHANGELOG.md)  on GitHub.
+
+# Getting Started
+
+To get started with Kubeflow we suggest following the steps below:
+
+
+1.  Follow our getting [started guide](https://www.kubeflow.org/docs/started/getting-started/)  to deploy Kubeflow 
+1.  Try out Kubeflow using one of the  existing models in the [examples repository](https://github.com/kubeflow/examples). 
+
+
+## What's Next
+
+Our next major release, 0.4, will be coming by the end of this year. Here are some key areas we are working on.
+
+
+##3 Ease of Use
+
+To make Kubeflow more accessible to data scientists, we want to make it possible to perform common ML tasks without having to learn Kubernetes. In 0.4, we hope to provide an initial version of tooling that makes it possible for data scientists to train and deploy models entirely from a notebook.
+
+
+### Model Tracking
+
+Keeping track of experiments is a major source of toil for data scientists and ML Engineers. In 0.4, we plan on making it easier to track models no matter how they were trained.
+
+
+### Production Readiness
+
+With 0.4 we want to continue to push Kubeflow towards production readiness by graduating our PyTorch and TFJob operators to beta.
+
+
+## Come Help!
+
+As always, we're listening! Please tell us the feature (or features) you'd really like to see that aren't there yet. Some options for making your voice heard include:
+
+
+*   The Kubeflow Slack channelThe GitHub repo
+*   The [Kubeflow-discuss](https://groups.google.com/forum/#!forum/kubeflow-discuss) email list
+*   The [Kubeflow Twitter](http://twitter.com/kubeflow) account
+*   Our [weekly community meeting](https://github.com/kubeflow/community)
+*   Please download and run [Kubeflow](https://github.com/kubeflow/kubeflow/pull/330/files), and submit bugs!
+
+Thank you for all your support so far!
+
+Jeremy Lewi, Abhishek Gupta, & Chris Cho (Google)

```

Comment:
Add emphasis around the authors similar to other blog posts we have.
*Jeremy Lewi, Abhishek Gupta, & Chris Cho (Google)*